nepali_datetime/__init__.py

```diff
@@ -702,6 +702,16 @@ def combine(cls, date, time):
             time.tzinfo
         )
 
+    @classmethod
+    def current_fiscal_year(cls):
+        """Return Current Fiscal Year"""
+        current_datetime = cls.now()
+        if current_datetime.month < 4:
+            current_fiscal_year = f"{current_datetime.year - 1}/{current_datetime.year}"
+        else:
+            current_fiscal_year = f"{current_datetime.year}/{current_datetime.year + 1}"
+        return current_fiscal_year
```

Comment:
@lalitmahato can we return the fiscal year in `tuple` format like `(2079, 2080)` so that developer calling it does not need to split it and format it according to his need?

Reply:
@amitgaru2 yeah, we can,
What if the developers directly need the fiscal year in the following formats in string form: ```2080/2081```, ```2080-2081```, ```2080/81```, ```2080-81```

Reply:
@lalitmahato please read my previous comment carefully and you're providing examples that is contradicting your own implementation.

Reply:
```@amitgaru2``` I am thinking of implementing both features,  returning ```tuple``` and returning ```str``` format as well. for now I ma closing this PR and reopen another PR within some days, which return fiscal year in ```tuple``` format and also return in ```str``` format (As per structure/format provided by the user).